src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -114,23 +125,6 @@ const DictionaryDetailCard = (props) => {
               Go to concepts
             </Link>
           </fieldset>
-          <fieldset>
```

Comment:
Why was this deleted? Was it part of the issue scope?

Reply:
@kmunene Thank you for making that observation. No, it was not part of this issue scope. As agreed by the team, I have removed those changes that were fixing a couple of bugs and I have created a ticket to specifically handle that here: https://issues.openmrs.org/browse/OCLOMRS-223